seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json

```diff
@@ -390,6 +390,117 @@
       "type": "backend",
       "challengeType": 0,
       "translations": {}
-    }  
+    },  
+    {
+      "id": "589a69f5f9fc0f352b528e6f",
+      "title": "Social Authentication with OAuth",
+      "description": [
+        [
+          "",
+          "",
```

Comment:
Why are you inserting two blank lines into the instructions?

Would a `<hr>` be better?